src/features/AuthTestPage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import NoticeTemplate from '../common/noticeTemplate/NoticeTemplate';
+
+const AuthTestPage = () => {
+  return <NoticeTemplate titleText="AUTH TEST" message="AUTH TEST" />;
+};
+
+export default AuthTestPage;
```

Comment:
I think we can already put the profile page behind a private route instead of having this test page.